lib/service/service.go

```diff
@@ -1824,7 +1825,7 @@ func (process *TeleportProcess) initAuthService() error {
 		if listener != nil {
 			warnOnErr(listener.Close(), log)
 		}
-		if payload == nil {
+		if ctx, _ := payload.(context.Context); ctx == nil {
```

Comment:
```suggestion
		if ctx := payload.(context.Context); ctx == nil {
```

Reply:
changed to use `payloadContext` in de047f3a0a